main.nf

```diff
@@ -648,56 +650,56 @@ def d4_coverage_version(task) {
 }
 
 process verifybamid2 {
-        cpus 16
```

Comment:
Looks like we are using tabs in the rest of the pipeline, but this was added with spaces